### PR TITLE
Remove 'MMDBG' prefix. Remove/downgrade some log levels.

### PR DESCRIPTION
**Pull request by InoMurko**
General cleanup.

 Changes to be committed:
	modified:   accounts/abi/bind/backends/simulated.go
	modified:   core/blockchain.go
	modified:   core/rawdb/accessors_chain.go
	modified:   core/state_processor.go
	modified:   core/state_transition.go
	modified:   core/types/rollup_l1_cost.go
	modified:   core/types/transaction.go
	modified:   core/types/transaction_signing.go
	modified:   eth/backend.go
	modified:   eth/stagedsync/stage_headers.go
	modified:   eth/stagedsync/stage_mining_create_block.go
	modified:   eth/stagedsync/stage_mining_exec.go
	modified:   eth/stagedsync/stagebuilder.go
	modified:   go.mod
	modified:   go.sum
	modified:   turbo/builder/block_builder.go
	modified:   turbo/engineapi/engine_helpers/request_list.go
	modified:   turbo/engineapi/engine_server.go
	modified:   turbo/jsonrpc/engine_api.go
	modified:   turbo/jsonrpc/eth_block.go
	modified:   turbo/jsonrpc/eth_receipts.go
	modified:   turbo/jsonrpc/send_transaction.go
	modified:   turbo/stages/stageloop.go

### CODE REVIEW

**Comment by InoMurko** [comment]
I just cleaned up https://github.com/bobanetwork/v3-erigon/pull/37

File: core/state_processor.go [hunk 1]
```diff
@@ -115,6 +117,24 @@ func applyTransaction(config *chain.Config, engine consensus.EngineReader, gp *G
...
+				feeScalar := new(big.Float).SetInt(scalar.ToBig())
+				receipt.FeeScalar = feeScalar.Quo(feeScalar, big.NewFloat(1e6))
+				l1GasUsed := uint256.NewInt(msg.RollupDataGas())
+				receipt.L1GasUsed = l1GasUsed.Add(l1GasUsed, &overhead).ToBig()
+				receipt.L1GasPrice = l1BaseFee.ToBig()
+				log.Debug("Set L1Fee for receipt", "fee", receipt.L1Fee, "feeScalar", feeScalar, "l1GasPrice", receipt.L1GasPrice, "l1GasUsed", receipt.L1GasUsed, "txhash", tx.Hash())
+			} else {
+				log.Warn("No L1 cost function set in context", "txhash", tx.Hash())
+			}
+			//Bedrock
```
Comment (boyuan-chen) [inline core/state_processor.go]: This logic can be removed

**Review by mmontour1306 — APPROVED** [review]

File: core/state_processor.go (hunk 1 above)
Comment (InoMurko) [reply core/state_processor.go]: let's remove this in a separate pr as I'm unsure of the consequences where this is used

Comment (boyuan-chen) [reply core/state_processor.go]: The logic of handling these special fields is added in this PR and the above code is removed. https://github.com/bobanetwork/v3-erigon/pull/30
I am not sure why this part of code is added back again.

Comment (InoMurko) [reply core/state_processor.go]: uff, you're right. my bad

**Review by boyuan-chen — APPROVED** [review]

File: eth/stagedsync/stage_headers.go [hunk 2]
```diff
@@ -383,9 +381,7 @@ func startHandlingForkChoice(
 	if err != nil {
 		return nil, err
 	}
-	log.Info("MMDBG Forking point is", "blockNumber", forkingPoint)
-	isBedrockBlock := cfg.chainConfig.Optimism != nil && cfg.chainConfig.IsBedrock(header.Number.Uint64())
-	if forkingPoint < preProgress && (isBedrockBlock || cfg.chainConfig.Optimism == nil) {
+	if forkingPoint < preProgress {
```
Comment (jyellick) [inline eth/stagedsync/stage_headers.go]: The condition for this 'if' has changed and no longer checks for bedrock/optimism

File: eth/stagedsync/stage_mining_create_block.go [hunk 3]
```diff
@@ -174,17 +174,9 @@ func SpawnMiningCreateBlockStage(s *StageState, tx kv.RwTx, cfg MiningCreateBloc
 	}
 
 	header := core.MakeEmptyHeader(parent, &cfg.chainConfig, timestamp, &cfg.miner.MiningConfig.GasLimit)
-	if cfg.blockBuilderParameters != nil && cfg.blockBuilderParameters.GasLimit != nil {
-		log.Info("MMDBG Override gas limit from Engine API", "old", header.GasLimit, "new", *cfg.blockBuilderParameters.GasLimit)
+	if cfg.blockBuilderParameters != nil && cfg.blockBuilderParameters.GasLimit != nil && header.GasLimit != *cfg.blockBuilderParameters.GasLimit {
```
Comment (jyellick) [inline eth/stagedsync/stage_mining_create_block.go:177]: The previous check ensured that the GasLimit was not nil before dereferencing -- are we risking a nil dereference there?

**Pull request merged**